packages/meta-extraction/src/abstractExtractionRules.js

```diff
@@ -144,11 +151,6 @@ const gatherDublinCoreTags = async (page) => {
       return { name, type, value };
     })
   );
-//   const evidences = await Promise.all(
-//     ['DC.Creator', 'DC.Subject', 'DC.Identifier', 'DC.Type'].map(
-//       async (name) => selectAllMetaEvidence(page, name)
-//     )
-//   );
```

Comment:
Can we delete this commented code?

Reply:
this commented code is deleted isn't it?

Reply:
OMG! I just wanted to write comments in your PR apparently.